### DISCUSSION TITLE
Due date as no. of days is not visually helpful

### DISCUSSION THREAD

**Discussion by DickinsonSa** [TechbeeAT/jtxBoard#375] [General]
When I create a future task, it says something like...

Due in 36 days

Without any visual clue as to when the task is due (ie a calendar), perhaps something like this might be more helpful... 

Due Friday 28th March 2023 (12 days) 

Can this be considered a possible future enhancement please?

**Comment by patrickunterwegs** [comment]
Hey @DickinsonSa , sure, you're absolutely right. What do you think about just showing the date when the date is more than - let's say - 7 days in the future?

**Reply by DickinsonSa** [reply]
Hi Patrick,
Sounds good to me.

**Comment by DickinsonSa** [comment]
Hi Patrick, just had a thought... good idea about only showing the date if the event is > 7 days but, just looking at my tasks and one says...

Due in 4 days

Now, I had to think... today is... (erm Friday) so that'll be due on (thinking again)... Tuesday - yes! And because I've got ocd, I'll double check it in my head. 

Can we enhance the message to simply say... 

Due on Tuesday

Anything over 7 days leave as you originally said.

**Reply by patrickunterwegs** [reply]
Hey @DickinsonSa , sounds like I nice idea :-) in this case I'd say whatever works best and let's try it ;-)
I anyway wanted to make this message shorter and replace the "Due in x days" by a symbol and text, so I would like to try it like that: "->| Monday"
similarly for the start date
"|-> Sunday"

This would just contain the same information but much shorter. I hope it would still be clear with symbols.

**Reply by DickinsonSa** [reply]
Look forward.
